src/posh-winfetch.ps1

```diff
@@ -369,7 +383,7 @@ function info_disk {
 # ===== POWERSHELL VERSION =====
 function info_pwsh {
     return @{
-        title   = "PowerShell"
+        title   = "Shell"
```

Comment:
This is a different topic, technically the `shell` will never be anything other than PowerShell, but "Shell" might be clearer keyword, so I'm unsure whether this should be changed?

Reply:
I changed it because other similar tools use the term 'Shell' too (like cmdfetch or neofetch). I know it'll always be PowerShell, but the line `PowerShell: PowerShell v7.1.0` just sounds redundant.

Reply:
I agree `Shell` is a more familiar name. I suppose it depends on if we want it to be a `Shell: PowerShell v7.1.0` field or a PowerShell version field `PowerShell: v7.1.0`.